### PR TITLE
Make env differentiation between index-files

### PR DESCRIPTION
**Pull request by Albertlarsen**
## Description
Make environment differentiation between index-files

## Related Issue(s)
- #390 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -1,26 +1,61 @@
...
+        import RefreshRuntime from "http://localhost:5173/@@react-refresh"
+        RefreshRuntime.injectIntoGlobalHook(window)
+        window.$RefreshReg$ = () => {}
+        window.$RefreshSig$ = () => (type) => type
+        window.__vite_plugin_react_preamble_installed__ = true
+        </script>
+        <script type="module" src="http://localhost:5173/@@vite/client"></script>
+        <meta charset="UTF-8"/>
+        <link rel="icon" type="image/svg+xml" href="/src/assets/altinn-logo.svg"/>
+        <!-- Font Inter support -->
```
Comment (allinox) [inline backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Views/Home/Index.cshtml:46]: Is it possible to have an env check only on the parts that need to change so we don't get so much duplicated code?

**Review by allinox — CHANGES_REQUESTED** [review]

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -1,26 +1,61 @@
...
-    <link rel="preconnect" href='https://github.com/Altinn/altinn-cdn'>
-    <link
-      rel="stylesheet"
-      href="https://altinncdn.no/fonts/inter/inter.css"
-    />
-    <title>Altinn</title>
-</head>
-<body>
+
+@if (isProd)
```
[the remark this answers is not in the document]
Comment (Albertlarsen) [reply backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Views/Home/Index.cshtml]: I agree

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -1,61 +1,44 @@
...
-        <meta name="description" content="Altinn API Delegering">
-        <meta name="keywords" content="Altinn Profil API Delegering">
-        <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-        <link rel="icon" href="https://altinncdn.no/favicon.ico">
-        <!-- Font Inter support -->
-        <link rel="preconnect" href='https://rsms.me/'>
-        <link rel="stylesheet" href='https://rsms.me/inter/inter.css'>
-        <title>Altinn</title>
-    </head>
+    }
```
Comment (allinox) [inline backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Views/Home/Index.cshtml:11]: Is this just an empty if?

**Review by allinox — CHANGES_REQUESTED** [review]

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Views/Home/Index.cshtml (hunk 3 above)
Comment (Albertlarsen) [reply backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Views/Home/Index.cshtml:11]: Woops 😅

**Review by allinox — APPROVED** [review]
Awesome job! 😁🌟
This will save us a lot of time going forward - thanks for taking the initiative to fix it! 🌻

**Pull request merged**